model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/events/GroupAddedEvent.java

```diff
@@ -69,6 +74,7 @@ public void writeObject(ObjectOutput output, GroupAddedEvent obj) throws IOExcep
 
             MarshallUtil.marshallString(obj.groupId, output);
             MarshallUtil.marshallString(obj.realmId, output);
+            MarshallUtil.marshallString(obj.parentId, output);
```

Comment:
This should be `VERSION_2` serialization, since the structure of the message has changed and it is thus not possible to read version 1 messages anymore without an exception.

Reply:
@hmlnarik Updated, Thanks! Doublechecked cluster tests on my laptop and the scenario from KEYCLOAK-13369 ajd still works fine.